python/lsst/source/injection/inject_engine.py

```diff
@@ -444,8 +453,38 @@ def inject_galsim_objects_into_exposure(
     bbox = exposure.getBBox()
     full_bounds = galsim.BoundsI(bbox.minX, bbox.maxX, bbox.minY, bbox.maxY)
     galsim_image = galsim.Image(exposure.image.array, bounds=full_bounds)
+    galsim_variance = galsim.Image(exposure.variance.array, bounds=full_bounds)
     pixel_scale = wcs.getPixelScale(bbox.getCenter()).asArcseconds()
 
+    variance_scale = galsim_variance.copy()
```

Comment:
Please add a brief in-line comment above this making clear how this array will be used below.

Reply:
This has now been re-labeled with the more descriptive `gain_map`.

Reply:
It looks like `gain_map` has now been removed entirely, following review.